.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
   - docker-compose -v
   - docker -v
   # symfony application folder
-  - mkdir symfony
+  - mkdir -p .data/src
```

Comment:
It's just a common practice to have a data volume, with the data in it. And a codebase is data. Since the database container will also create a `.data/db` folder, it's common to have some kind of source folder (for development). In production this can never run smoothly when using scaled containers. Because the codebase have to be installed on each and every host! 
In production you might want to use a custom file-container which needs to be rebuild every release, or named volumes which can be local storage, cloud storage, NFS storage, etc. 

Also when data sources differ (EG: the codebase is local and database-data on cloud) multiple named volumes can be created. When using in production, for example deploying with Rancher, aliases can be made for containers. So that a database can also be a remote service. 

Also when installing or developing your symfony application with this approach, the container stack will not be affected. 


But if you are really sure you want a custom `symfony` folder for the source-code, I will revert that change.